source/authentication/email-password.txt

```diff
@@ -314,68 +310,63 @@ password reset email:
   particular template using an external service.
 
 - Interface with a MongoDB Atlas collection to implement a password
-  reset "cooldown period" to prevent too many password reset
+  reset "cooldown period". This can prevent too many password reset
   attempts on a single account in a particular time range.
 
 - Send custom password reset messages through a service other than
   email.
 
-Configure the following settings to have {+service-short+} automatically run a
-password reset function:
+{+service-short+} can automatically run a password reset function.
+Configure the following settings:
 
 - :guilabel:`Function`: the function run when an instance of the client
-  SDK calls the ``callResetPasswordFunction`` function. This function
-  must return an object containing a ``status`` key mapping to a string
+  SDK calls ``callResetPasswordFunction``. This function
+  must return an object containing a ``status`` key. The key maps to a string
   with one of the following values: ``success``, ``pending``, or
   ``fail``.
 
 .. note::
 
-   If you configure the provider to use a password reset function then
-   the client SDK ``sendResetPasswordEmail()`` function will return an
-   error. You may still need to call ``resetPassword()`` if the password
-   reset function returns a ``pending`` status.
+   Using a password reset function causes the client SDK ``sendResetPasswordEmail()`` 
+   function to return an error. The password reset function may return a ``pending`` status.
+   If so, you may still need to call ``resetPassword()``.
 
 The custom password reset function signature is
 :wikipedia:`variadic <Variadic_function>`: it accepts any number of
-parameters. The first of these parameters is always an object containing
-user data and confirmation tokens. All parameters after the first are
-custom parameters passed into the Client SDK as an argument collection.
-For instance, the Client SDK call:
+parameters. The first is always an object containing user data and confirmation tokens.
+All following parameters are custom parameters. They are passed into the Client SDK 
+as an argument collection. For instance, the Client SDK call:
 
 .. code-block:: javascript
 
    callResetPasswordFunction("myUsername", "newPassword", ["Security Question Answer 1", "Security Question Answer 2", "securityCode:0510"])
 
 
-Would be translated into the following custom password reset function
-signature:
+Becomes this customized signature for the password reset function:
 
 .. code-block:: javascript
 
    resetFunc({username, password, token, tokenId}, securityAnswer1, securityAnswer2, securitySMSCode)
 
-Each element of the custom argument array is spread into a parameter for
-the custom password reset function, starting with the second parameter.
+There are multiple elements of the custom argument array. Starting with the second parameter,
+ these elements are passed into the password reset function. 
```

Comment:
I think there's some Snooty shenanigans going on here. The space being the leading character on line 366 seems to mean it's getting cut off entirely, as it renders without a space after the comma on the staging build:
> Starting with the second parameter,these elements are passed into the password reset function.

Maybe delete the space at the beginning of line 366 and add one after the comma in line 365?